src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java

```diff
@@ -94,16 +97,20 @@ public class NetworkConversionService {
 
     private final NetworkConversionExecutionService networkConversionExecutionService;
 
+    private final NotificationService notificationService;
+
     private final ObjectMapper objectMapper;
 
     @Autowired
     public NetworkConversionService(@Value("${backing-services.case-server.base-uri:http://case-server/}") String caseServerBaseUri,
                                     @Value("${backing-services.geo-data-server.base-uri:http://geo-data-server/}") String geoDataServerBaseUri,
                                     @Value("${backing-services.report-server.base-uri:http://report-server}") String reportServerURI,
-                                    NetworkStoreService networkStoreService, @Lazy EquipmentInfosService equipmentInfosService, NetworkConversionExecutionService networkConversionExecutionService) {
+                                    NetworkStoreService networkStoreService, @Lazy EquipmentInfosService equipmentInfosService, NetworkConversionExecutionService networkConversionExecutionService,
```

Comment:
Could we take the chance to insert a newline before `@Lazy` and between the other non terminal comma and the next N ?

Reply:
fixed